src/personalAccessTokens/tokenCard/NewToken.component.jsx

```diff
@@ -17,6 +18,10 @@ const NewToken = ({ token }) => {
     return (
         <div>
             <h3 className={styles.header}>{i18n.t('Newly created token')}</h3>
+            <div className={styles.item}>
```

Comment:
it might be good to conditionally display this item, as if you have a non-patched version of the backend, you will create the token without the code and then be be brought to this view where it will say `Token name` with noting next to it